pkg/app/app.go

```diff
@@ -148,7 +148,7 @@ func isGitVersionValid(versionStr string) bool {
 
 func isDirectoryAGitRepository(dir string) (bool, error) {
 	info, err := os.Stat(filepath.Join(dir, ".git"))
-	return info != nil && info.IsDir(), err
```

Comment:
Are we removing this because the path may not actually be a directory? If so we should change the function / arg names

Reply:
The `arg` is the path to the repo which is definitely a `dir`.

The `<dir>/.git` may be a:
- directory ( ordinary git repository )
- file ( an additional worktree git repository )

It's a `repo` whichever way you look at it though.

Reply:
ah cool, that makes sense